### DISCUSSION TITLE
MIKEIO and anaconda

### DISCUSSION THREAD

**Discussion by sudheesh-k** [DHI/mikeio#703] [General]
How to install MIKEIO with conda

**Comment by ecomodeller** [comment]
Install using pip. You can mix conda packages and pip packages in the same conda environment. https://docs.anaconda.com/free/working-with-conda/packages/install-packages/

**Reply by sudheesh-k** [reply]
But it saying that " Don't use conda to install MIKE IO!, the version on conda is outdated." (https://github.com/DHI/mikeio?tab=readme-ov-file#installation)

**Reply by jsmariegaard** [reply]
Yes, on a general level you can mix conda packages and pip packages, but for MIKE IO only use pip :-)

**Comment by jsmariegaard** [comment]
See also the comment about conda in https://github.com/DHI/mikeio?tab=readme-ov-file#installation

**Reply by sudheesh-k** [reply]
But it saying that " Don't use conda to install MIKE IO!, the version on conda is outdated." (https://github.com/DHI/mikeio?tab=readme-ov-file#installation)

**Reply by jsmariegaard** [reply]
Exactly
